test/acceptance/features/bindAppToServiceAnnotationsOptional.feature

```diff
@@ -0,0 +1,173 @@
+            spec:
+                uri: "youknow.where"
+                image: "busybox"
+            """
+        When Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: $scenario_id
+            spec:
+                services:
+                  - group: stable.example.com
+                    version: v1
+                    kind: AppConfig
+                    name: $scenario_id
+                application:
+                    group: apps
+                    version: v1
+                    resource: deployments
+                    name: $scenario_id
+            """
+        Then Service Binding is ready
+        * Content of file "/bindings/$scenario_id/uri" in application pod is
+            """
+            youknow.where
+            """
+        * Content of file "/bindings/$scenario_id/image" in application pod is
+            """
+            busybox
+            """
+
+    Scenario: Bind to the application with annotations and with missing optional source field to be excluded
+        Given Generic test application is running
+        And The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: AppConfig
+            metadata:
+                name: $scenario_id
+                annottions:
+                    service.binding/uri: "path={.spec.uri},optional=exclude"
+                    service.binding/image: "path={.spec.image}"
+            spec:
+                image: "busybox"
+            """
+        When Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: $scenario_id
+            spec:
+                services:
+                  - group: stable.example.com
+                    version: v1
+                    kind: AppConfig
+                    name: $scenario_id
+                application:
+                    group: apps
+                    version: v1
+                    resource: deployments
+                    name: $scenario_id
+            """
+        Then Service Binding is ready
+        * File "/bindings/$scenario_id/uri" is unavailable in application pod
+        * Content of file "/bindings/$scenario_id/image" in application pod is
+            """
+            busybox
+            """
+
+    Scenario: Bind to the application with annotations and with missing optional source field to be projected as empty
+        Given Generic test application is running
+        And The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: AppConfig
+            metadata:
+                name: $scenario_id
+                annottions:
+                    service.binding/uri: "path={.spec.uri},optional=empty"
```

Comment:
The value for `optional` should be `true` or `false` as explained in the story.

Reply:
+1